CHANGELOG.md

```diff
@@ -5,12 +5,16 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes:
+- Updated dependencies for PureScript 0.14 (#80)
 
 New features:
 
 Bugfixes:
 
 Other improvements:
+- Migrated CI to GitHub Actions and updated installation instructions to use Spago (#81)
+- Added a CHANGELOG.md file and pull request template (#82 #83)
+- Replaced unsafe FFI functions with `unsafeCoerce` (#72)
```

Comment:
Perhaps we should call out that we specifically replaced `unsafeToForeign` and `unsafeFromForeign`

Reply:
Also, can we comma-separate multiple PRs? As in `(#82, #83)`?